### DISCUSSION TITLE
How i run rocket.chat using docker compose locally? for development

### DISCUSSION THREAD

**Discussion by MaHDiaLaGaB** [RocketChat/Rocket.Chat#31120] [Q&A]
I am working on changing some themes and adding AI agents to do some work for me using Langchain, but I want to build and run it locally using docker, and docker-compose.
i need clear steps because i found like 6 Dockerfiles and i got lost, the AI features i need it to be for Enterprise version that's why i want to run all the Microservices

**Comment by Spirizeon** [comment]
Hi, you can check these docs https://docs.rocket.chat/v1/docs/deploy-with-docker-docker-compose, lemme know these are what you're looking for.

**Comment by reetp** [comment]
The correct places to ask these questions are here:

https://open.rocket.chat/channel/support/
https://forums.rocket.chat/
